GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -278,7 +279,7 @@ private void LayoutRevisionInfo()
             if (AppSettings.ShowRevisionInfoNextToRevisionGrid.ValueOrDefault)
             {
                 RevisionInfo.Parent = RevisionsSplitContainer.Panel2;
-                RevisionsSplitContainer.SplitterDistance = RevisionsSplitContainer.Width - 500;
+                RevisionsSplitContainer.SplitterDistance = RevisionsSplitContainer.Width - 420;
```

Comment:
It will be scaled based on the current font size. 
If I am not mistaken then the current dpi aware strategy is that GitExt renders normally and users set a bigger font size to make the UI larger.